{
    "has_execution_error": false,
    "execution_output": "Traceback (most recent call last):\n  File \"/home/ql98/calendar-planning/source/../output/SMT/Qwen/Qwen2.5-Coder-32B-Instruct/meeting/n_pass/meeting_planning_example_432/5_pass/solution.py\", line 103, in <module>\n    current_friend = visit_map[visit_order[i]]\n                     ~~~~~~~~~^^^^^^^^^^^^^^^^\nKeyError: visit_order_0\n",
    "pred": {
        "itinerary": [
            {
                "action": "meet",
                "person": "David",
                "start_time": "13:00",
                "end_time": "14:00"
            },
            {
                "action": "meet",
                "person": "Alice",
                "start_time": "15:30",
                "end_time": "16:00"
            }
        ]
    },
    "gold": {
        "itinerary": [
            {
                "action": "meet",
                "person": "Joseph",
                "start_time": "09:24",
                "end_time": "10:54"
            },
            {
                "action": "meet",
                "person": "Kevin",
                "start_time": "11:16",
                "end_time": "11:46"
            },
            {
                "action": "meet",
                "person": "Barbara",
                "start_time": "12:03",
                "end_time": "12:18"
            },
            {
                "action": "meet",
                "person": "Jeffrey",
                "start_time": "17:30",
                "end_time": "18:30"
            }
        ]
    },
    "status": "Wrong plan",
    "violated_constraint": {
        "num_people_to_meet": 4
    },
    "is_exact_match": false,
    "constraints_satisfied": false,
    "pass_number": 5
}